index.bs

```diff
@@ -3922,6 +3936,35 @@ Note: The {{UserVerificationRequirement}} enumeration is deliberately not refere
 </div>
 
 
+### User-agent Hints Enumeration (enum <dfn enum>PublicKeyCredentialHints</dfn>) ### {#enum-hints}
+
+<xmp class="idl">
+    enum PublicKeyCredentialHints {
+        "security-key",
+        "local-device",
```

Comment:
As discussed during WAWG today we should probably replace `"local-device"` with `"client-device"` since the latter is defined under Terminology, and covers the case of RDP-assisted WebAuthn use that local device won't.